manifests/b/Bazarr/bazarr/1.0.2/Bazarr.bazarr.locale.en-US.yaml

```
@@ -0,0 +1,27 @@
+# Created with YamlCreate.ps1 v2.0.4 $debug=NVS1.5-1-19041-1320
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.locale.1.1.0.schema.json
+
+PackageIdentifier: Bazarr.bazarr
+PackageVersion: 1.0.2
+PackageLocale: en-US
+Publisher: morpheus65535
```

```suggestion
Publisher: Bazarr
```
This must match Control Panel.